testing/tests/DevExpress.ui.widgets.pivotGrid/pivotGrid.tests.js

```diff
@@ -5819,16 +5819,69 @@ QUnit.module('Vertical headers', {
     });
 
     // T696415
-    QUnit.test('Set column width with floating point', function(assert) {
-    // arrange
-        const headersArea = createHeadersArea(null, true);
-
-        headersArea.render($('#pivotArea'), this.data);
+    QUnit.test('headers and data columns has same width', function(assert) {
```

Comment:
Add the 'zoom' or 'hight dpi' word to the test name.

Reply:
No, it's old test. It's check the data and column width not directly related to a T914454.
It fix the T696415